src/_macosx.m

```diff
@@ -1000,74 +992,24 @@ -(void)zoom:(id)sender
 }
 
 -(void)configure_subplots:(id)sender
-{   PyObject* canvas;
-    View* view;
-    PyObject* size;
-    NSRect rect;
-    int width, height;
+{
+    PyObject *mod = NULL, *base_class = NULL, *ret = NULL;
```

Comment:
Sorry for the way-too-complicated original patch; actually, because the method has not been overridden on Python's side, you can even directly call it, I think:
```objc
{   PyObject* result;
    PyGILState_STATE gstate;
    gstate = PyGILState_Ensure();
    result = PyObject_CallMethod(toolbar, "configure_subplots", "");
    if(result)
        Py_DECREF(result);
    else
        PyErr_Print();
    PyGILState_Release(gstate);
}
```
i.e. the same implementation as home, back, etc.  (I plan to merge all of these together in a single helper, in a later PR.)

Reply:
I believe you're right; it seems to work on the test Mac anyway.